### PR TITLE
Added and updated the content for cookie and accessibility pages

### PR DESCRIPTION
**Pull request by dhvander**
Updated the tag colour for non prod environments (stage and prod have same tag)

### CODE REVIEW

File: request_a_govuk_domain/request/utils.py [hunk 1]
```diff
@@ -366,13 +366,15 @@ def phase_header_text(_request):
     :return:
     """
     phase = os.getenv("ENVIRONMENT")
-    if phase != "prod":
+    if phase not in ["prod", "stage"]:
         return {
+            "PHASE_COLOUR": "govuk-tag--pink",
```
Comment (maxf) [inline request_a_govuk_domain/request/utils.py:371]: I think `PHASE_CLASS` is better as it's clearer what it is and could be something other than a colour (bold or italic)

**Review by maxf — CHANGES_REQUESTED** [review]

File: request_a_govuk_domain/request/utils.py (hunk 1 above)
Comment (dhvander) [reply request_a_govuk_domain/request/utils.py]: done and also updated the terms and the links

File: request_a_govuk_domain/request/templates/cookies.html [hunk 2]
```diff
@@ -1,159 +1,85 @@
 {% extends "base.html" %}
 {% load govuk_frontend_django %}
 
-{% block title %}Cookies - Get approval to use a .gov.uk domain name - GOV.UK{% endblock %}
+{% block title %}Cookies on the Get approval to use a .gov.uk domain name service{% endblock %}
```
Comment (maxf) [inline request_a_govuk_domain/request/templates/cookies.html:4]: The previous title was correct. Should always be in 3 parts, like all other services and pages.

File: request_a_govuk_domain/request/templates/cookies.html [hunk 3]
```diff
@@ -1,159 +1,85 @@
...
 
 {% block content %}
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
```
Comment (maxf) [inline request_a_govuk_domain/request/templates/cookies.html:8]: Can you keep the indentation as before (2 chars) and not use tabs?

**Review by maxf — CHANGES_REQUESTED** [review]

**Review by maxf — APPROVED** [review]

**Pull request merged**